slk/repl/repl.py

```diff
@@ -1343,10 +1734,20 @@ def help_subscribe(self: SidechainRepl) -> None:
     ##################
     # quit
     def do_quit(self: SidechainRepl, line: str) -> bool:
+        """
+        Implementation of the `quit` REPL command.
+
+        Args:
+            line: The command-line arguments.
+
+        Returns:
+            True, so that the REPL closes.
```

Comment:
True when the REPL closes successfully.

Reply:
Returning `True` in a method kills the REPL, not the other way around.